docs/guides/meta/meta-avatars-setup.md

```diff
@@ -2,13 +2,15 @@
 layout: docs
 title: Meta Avatars Setup
 ---
-import doubleAvatarWave from './meta-avatars/double-avatar-wave.mp4'
+import avatarWaveGroup from './meta-avatars/avatar-wave-group.mp4'
 import metaXR from './meta-avatars/install-meta-xr.mp4'
-import metaSample from './meta-avatars/install-meta-sample.mp4'
-import avatarWave from './meta-avatars/avatar-wave.mp4'
+import packageDependencies from './meta-avatars/install-package-dependencies.mp4'
+import sceneSetup from './meta-avatars/scene-setup.mp4'
+import avatarThumbsUp from './meta-avatars/avatar-thumbs-up.mp4'
 
 # Meta Avatars
-<video width="100%" controls><source src={doubleAvatarWave} /></video>
+<video width="100%" controls><source src={avatarWaveGroup} /></video>
```

Comment:
This video has a mouse cursor in it. Also it's really shaky. Are you able to record this via Unity like the original one was? I know Quest and Rift apps aren't compatible, but are you able to patch your build to just load the correct avatar design rather than trying to load it from their Oculus account? iirc that's what we did last time.